ghost/admin/app/components/posts/debug/email-error-message.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+import {action} from '@ember/object';
+import {tracked} from '@glimmer/tracking';
+
+export default class EmailErrorMessage extends Component {
+    @tracked more = false;
+
+    @action
+    setup(element) {
+        const errorContainer = element.querySelector('span');
+        if (element.offsetWidth <= errorContainer.offsetWidth) {
+            this.more = true;
+        }
+    }
+}
```

Comment:
We should ideally have an extra line in the end of the files , not a big deal tho :)